concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+from logging import getLogger
+import os
+import sys
+
+import django
+from django.db import transaction
+
+from common.helpers import deserialize_message
+
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    with transaction.atomic(using='control'):
+        for subtask in Subtask.objects.select_for_update().all():
+            try:
+                report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+                if not isinstance(report_computed_task.size, int):
+                    logger.info(f'Size inside ReportComputedTask message inside subtask with subtask_id: {subtask.subtask_id} is: {report_computed_task.size}')
+                    delete_unsupported_messages(subtask)
+            except Exception as exception:
+                logger.info(f'During message deserialization exception raised: {exception}')
+                delete_unsupported_messages(subtask)
```

Comment:
Why are you ignoring the error even though you don't even know what kind of error it is?

When you get an unknown error you should fail, not silence it and try to force the deletion. If you do it like this, you will delete any message that fails to decode which is definitely not something we want to do without first checking why.